data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -1225,6 +1225,105 @@ def query_template(table_era):
 res2
 
 # # final summary result
+# -
+
+
+# # Q17 Wear study table
+#
+# DC-3340
+#
+# This check confirms that the wear_s.tudy table contains:
+# 1. Only one row per participant
+# 2. Wear study participants are also found in the CDR person table.
+# 3. Wear study participants have primary consent records in observation.
+# 4. Wear study start dates coincide with records in observation. Date shifted in RT.
```

Comment:
I like the idea of moving this check. It would be cleaner for sure. It would not rely on joining back to the obs table where the data might not exist.